site/docs/reference/Connectors/capture-connectors/OracleDB/OracleDB.md

```diff
@@ -0,0 +1,171 @@
+# OracleDB
+This connector captures data from OracleDB into Flow collections using [Oracle Logminer](https://docs.oracle.com/en/database/oracle/oracle-database/19/sutil/oracle-logminer-utility.html#GUID-2555A155-01E3-483E-9FC6-2BDC2D8A4093).
+
+It is available for use in the Flow web application. For local development or open-source workflows, `ghcr.io/estuary/source-oracle:dev` provides the latest version of the connector as a Docker image. You can also follow the link in your browser to see past image versions.
+
+## Prerequisites
+* Oracle 11g or above
+* Allow connections from Estuary Flow to your Oracle database (if they exist in separate VPCs)
+* Create a dedicated read-only Estuary Flow user with access to all tables needed for replication
+
+## Setup
+Follow the steps below to set up the OracleDB connector.
+
+### Create a Dedicated User
+
+Creating a dedicated database user with read-only access is recommended for better permission control and auditing.
+
+1. To create the user, run the following commands against your database:
+
+```sql
+CREATE USER estuary_flow_user IDENTIFIED BY <your_password_here>;
+GRANT CREATE SESSION TO estuary_flow_user;
+```
+
+2. Next, grant the user read-only access to the relevant tables. The simplest way is to grant read access to all tables in the schema as follows:
+
+```sql
+GRANT SELECT ANY TABLE TO estuary_flow_user;
+```
+
+3. Alternatively, you can be more granular and grant access to specific tables in different schemas:
+
+```sql
+GRANT SELECT ON "<schema_a>"."<table_1>" TO estuary_flow_user;
+GRANT SELECT ON "<schema_b>"."<table_2>" TO estuary_flow_user;
+```
+
+4. Create a watermarks table:
+```sql
+CREATE TABLE estuary_flow_user.FLOW_WATERMARKS(SLOT varchar(1000) PRIMARY KEY, WATERMARK varchar(4000));
+```
+
+5. Finally you need to grant the user access to use logminer, read metadata from the database and write to the watermarks table:
+
+```sql
+GRANT SELECT_CATALOG_ROLE TO estuary_flow_user;
+GRANT EXECUTE_CATALOG_ROLE TO estuary_flow_user;
+GRANT SELECT ON V$DATABASE TO estuary_flow_user;
+GRANT SELECT ANY TABLE TO estuary_flow_user;
+GRANT SELECT ON V$LOG TO estuary_flow_user;
+GRANT LOGMINING TO estuary_flow_user;
+
+GRANT INSERT, UPDATE ON estuary_flow_user.FLOW_WATERMARKS TO estuary_flow_user;
+```
+
+5. Enable supplemental logging:
+
+For normal instances use:
+```sql
+ALTER DATABASE ADD SUPPLEMENTAL LOG DATA (ALL) COLUMNS;
+```
+
+For Amazon RDS instances use:
+```sql
+BEGIN rdsadmin.rdsadmin_util.alter_supplemental_logging(p_action => 'ADD', p_type   => 'ALL'); end;
+```
+
+6. Create a watermarks table and grant permis
```

Comment:
Line is cut off, also there were already instructions to create the watermarks table above